model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_manager.py

```diff
@@ -106,6 +121,21 @@ def get_sensitivity_metric(self) -> Callable:
 
         return self.sensitivity_evaluator.compute_metric
 
+    def _calc_ru_fn(self, ru_target, ru_fn, mp_cfg):
+        """
+        Computes a resource utilization for a certain mp configuration
+        The method computes a resource utilization vector for specific target resource utilization.
+
+        Returns: resource utilization value.
+
+        """
+        # ru_fn is a pair of resource utilization computation method and
+        # resource utilization aggregation method (in this method we only need the first one)
+        if ru_target is RUTarget.ACTIVATION:
```

Comment:
maybe add a todo to eliminate this condition. this requires a refactor in the search manager but we need to include this in the refactor

Reply:
This appears in multiple places. I'll add the TODO in the function itself.